server/database/queries/getFood/index.js

```diff
@@ -0,0 +1,3 @@
+const connection = require('../../config/connection');
+
+module.exports = () => connection.query('SELECT * FROM food');
```

Comment:
Sorry that I'm late to this PR, but I was just wondering - are users able to add new food types to the `food` table? Or is it just developers / admins that can add food types?
What I'm getting at is, will this table ever grow to some extreme length like 100,000 rows? Does the code need to consider that possibility, so that in that unlikely event, this query doesn't become slow?